store/sqlstore/file_info_store.go

```diff
@@ -95,6 +95,72 @@ func (fs SqlFileInfoStore) Get(id string) (*model.FileInfo, *model.AppError) {
 	return info, nil
 }
 
+func (fs SqlFileInfoStore) GetWithOptions(page, perPage uint, opt *model.GetFileInfosOptions) ([]*model.FileInfo, *model.AppError) {
+	if perPage == 0 || page == 0 {
+		return nil, nil
+	}
+
+	// Nil option means sort by create at ascending
```

Comment:
nit: this comment seems misplaced. We might have been given a `&model.GetFileInfosOptions{}` for which the sort is still undefined but this code block doesn't deal with that. Suggest we remove and leave to the later code.
```suggestion
```

Reply:
The intent was to say that a zero-value options struct, passed in either as a nil pointer or a pointer to an zero-value struct, is not an error case and sensible defaults are applied. I guess this is evident from the code so dropping the comment sounds good to me.